src/source/Common/Include_i.h

```diff
@@ -13,9 +13,15 @@ extern "C" {
 ////////////////////////////////////////////////////
 // Project include files
 ////////////////////////////////////////////////////
+#if defined(KVS_USE_OPENSSL)
 #include <openssl/sha.h>
 #include <openssl/hmac.h>
 #include <openssl/evp.h>
+#elif defined(KVS_USE_MBEDTLS)
+#include "mbedtls/sha256.h"
+#include <mbedtls/md.h>

```

Comment:
minor: shouldn't both be <mbedtls/*.h>?

Reply:
Using wildcards is a bad idea

Reply:
Didn't mean wildcard, I meant, shouldn't it be both 
#include "mbedtls/sha256.h"
#include "mbedtls/md.h"
or 
#include <mbedtls/sha256.h>
#include <mbedtls/md.h>

Reply:
Oh, I see. Yeah, I think angle brackets are the right way to go here